### ISSUE TITLE
Empty alt="" for the Featured Image block

### ISSUE THREAD

**Issue by huubl** [WordPress/gutenberg#57893]
## What problem does this address?

In the current implementation of the Featured Image block, there is no option to leave the alt attribute empty (alt=""). This can be a problem in cases where the image is purely decorative and does not contain information that is useful to the understanding of the webpage. In these cases, an empty alt text should be provided (alt="") so screen readers can ignore the image.

## What is your proposed solution?

Users should have the ability to assign an empty alt attribute when adding a Featured Image block. If the alt attribute is assigned as empty, the HTML output should reflect alt="".

**Comment by carolinan** [comment]
Hi
It is possible, all you need to do is remove the alt in the media library.

Remember that linked featured images are not decorative. For these links, the alt text is the link text.

**Comment by huubl** [comment]
> Hi It is possible, all you need to do is remove the alt in the media library.
> 
> Remember that linked featured images are not decorative. For these links, the alt text is the link text.

Yes, I know. But an image can be used in different contexts and can be decorative in some of them. In such cases, it's appropriate to have an empty alt attribute. This is why the proposed feature to allow an empty alt attribute in the Featured Image block could be beneficial. It would provide more flexibility to handle different use cases and improve the overall accessibility of the site.
